src/buffer/channel.rs

```diff
@@ -123,15 +123,18 @@ pub fn view<'a>(
     let topic = topic(state, clients, users, config);
 
     let text_input = show_text_input.then(|| {
-        column![input_view::view(
-            &state.input_view,
-            buffer,
-            users,
-            channels,
-            input_history,
-            is_focused
-        )
-        .map(Message::InputView)]
+        column![
+            vertical_space(4),
+            input_view::view(
```

Comment:
Why are we adding this? The other buffers (Server and Query) don't add additional space here

Reply:
Edit: I just saw comment https://github.com/squidowl/halloy/pull/237#discussion_r1516813482, will reply there